JShellAPI/src/main/resources/application.yaml

```diff
@@ -20,6 +20,9 @@ jshellapi:
   dockerResponseTimeout: 60
   dockerConnectionTimeout: 60
 
+  # JShellWrapper related
+  jshellwrapper-imageName: ${jshellWrapperImageName}
```

Comment:
Please follow the conventions : camelCase.

Reply:
I wanted to keep `jshellwrapper` like the top level property `jshellapi` which is defined lowercase

Reply:
Oh yeah, well, jshellapi should have been called jshell-api, so we will fix this later.

Reply:
Still not fixed